pandas/core/common.py

```diff
@@ -372,6 +372,26 @@ def intersection(*seqs):
     return type(seqs[0])(list(result))
 
 
+def _as_1d_array(values, dtype='object'):
```

Comment:
move to pandas/core/dtypes/cast.py (at the bottom); rename in that style as well.

can move _asarray_tuplesafe as well (here or future PR)